adapters/onetag/params_test.go

```diff
@@ -0,0 +1,51 @@
+package onetag
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestValidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range validParams {
+		if err := validator.Validate(openrtb_ext.BidderOneTag, json.RawMessage(p)); err != nil {
+			t.Errorf("Schema rejected valid params: %s", p)
+		}
+	}
+}
+
+func TestInvalidParams(t *testing.T) {
+	validator, err := openrtb_ext.NewBidderParamsValidator("../../static/bidder-params")
+	if err != nil {
+		t.Fatalf("Failed to fetch the json schema. %v", err)
+	}
+
+	for _, p := range invalidParams {
+		if err := validator.Validate(openrtb_ext.BidderOneTag, json.RawMessage(p)); err == nil {
+			t.Errorf("Schema allowed invalid params: %s", p)
+		}
+	}
+}
+
+var validParams = []string{
+	`{	"pubId": "386276e072",
+		"ext": {
+			"key1": "value1",
+			"key2": "value2"
+		}
+	}`,
+	`{"pubId": "386276e072"}`,
+}
+
+var invalidParams = []string{
+	`{"ext": {
+		"key1": "value1",
+		"key2": "value2"
+	}`,
```

Comment:
Please consider adding another the following cases here as well:
1. A case with empty params i.e. with neither `ext` or `pubId` defined
2. A case with `pubId` defined to be an empty string
3. A case with `pubId` defined to be a number rather than a string

Reply:
I added this test cases.